src/ColumnSelectionStep/ColumnSelectionStep.tsx

```diff
@@ -242,15 +242,9 @@ export const ColumnSelectionStep: FunctionComponent<ColumnSelectionStepProps> =
           </tbody>
         </table>
       </div>
-      <div className="ColumnSelectionStep-reserved-space">
-        {/* Render next step */}
-        {bucketColumns.length !== 0 && (
-          <>
-            <Divider />
-            {children({ bucketColumns, countInput: hasAidColumn ? countInput : 'Rows' })}
-          </>
-        )}
-      </div>
+      {/* Render next step */}
+      <Divider />
+      {children({ bucketColumns, countInput: hasAidColumn ? countInput : 'Rows' })}
```

Comment:
Shame that we overwhelm the user with 4 blue nav steps as soon as they load a file. Maybe we should re-think the "steps" idea.

Reply:
Hm, and what is the intended use of the global aggregation? Is it
1/ actually allow to release **anonymized** counts of either rows or entities
2/ just give an overview of what kind of data is handled?

If (2/) then maybe these two values should be presented on the `Data Preview` step, without anonymization?

Reply:
Yes, not ideal. The alternative would be to make the user press "Next" after each step. This would make first runs less overwhelming, but it is probably pretty annoying in the long run.
Let's postpone this discussion until after the first full release.

Reply:
> Hm, and what is the intended use of the global aggregation?

I think it provides an overview over the input data and over the effect anonymization has on it.

Reply:
> > Hm, and what is the intended use of the global aggregation?
> 
> I think it provides an overview of the input data and of the effect anonymization has on it.

Yeah, my point being here - is the anonymization of the entire table interesting that often, except pathological cases where the table has very few rows/entities?

If this effect is sometimes, but rarely interesting, we could have non-anonymized counts as part of the `Data Preview` and then if anyone wants the global anonymized aggregates, there could be a special `None` switch in the `Column Selection` step, which would need to be explicitly switched on to give the user the global anonymized aggregates.

Reply:
I find it interesting enough and Paul also seems to agree (he asked for it).
I don't like branches or special cases in the usage flow. The current approach has the big advantage that is consistent: the anonymization step runs the same regardless if you select something or not in the previous step.